src/lang/modifyAst/addFillet.ts

```diff
@@ -65,55 +62,13 @@ export function addFillet(
     sketchSegmentNode.callee.name
   )
   if (err(taggedSegment)) return taggedSegment
-  const { tag } = taggedSegment as { modifiedAst: Program; tag: string }
+  const { tag } = taggedSegment
 
   /**
    * Find Extrude Expression automatically
    */
 
   // 1. Get the sketch name
-  const pathToSketch = [pathToSegmentNode[0], pathToSegmentNode[1]]
-  const sketchNode = getNodeFromPath(_node, pathToSketch) as {
-    node: VariableDeclaration
-  }
-  const sketchName = sketchNode.node.declarations[0].id.name
-
-  // 2. Find the extrue expression with the same name
-  // TODO: Has to work with Pipe Expressions, Lofts, etc...
-
-  let extrudeRange: [number, number] | undefined = undefined
-  _node.body.forEach((node) => {
-    traverse(node, {
-      enter(node) {
-        if (node.type === 'CallExpression') {
-          if (node.callee.name === 'extrude') {
-            if (
-              node.arguments[1].type === 'Identifier' &&
-              node.arguments[1].name === sketchName
-            ) {
-              extrudeRange = [node.start, node.end]
-            }
-          }
-        }
-      },
-    })
-  })
-
-  if (extrudeRange === undefined) {
-    return new Error('No valid body found')
-  }
-
-  const preselectedPathToExtrudeTag = getNodePathFromSourceRange(
-    _node,
-    extrudeRange
-  )
-  const preselectedPathToExtrudeNode = [
-    preselectedPathToExtrudeTag[0],
-    preselectedPathToExtrudeTag[1],
-  ]
-  if (pathToExtrudeNode.length < 2) {
-    pathToExtrudeNode = preselectedPathToExtrudeNode
-  }
```

Comment:
There was no point in having the `pathToExtrudeNode` as an input if we were not going to use it, instead I move the logic for finding this path to node into the `AST fillet` action. And the way I got the node was with the same method we use for find the extrude for sketch on face, to keep it consistent.